syft/generic/pointers/object_pointer.py

```diff
@@ -343,6 +343,23 @@ def __del__(self):
             if self.point_to_attr is None:
                 self.owner.send_msg(ForceObjectDeleteMessage(self.id_at_location), self.location)
 
+    def force_delete(self):
```

Comment:
I guess [this `__del__`](https://github.com/OpenMined/PySyft/blob/master/syft/generic/pointers/object_pointer.py#L329) method does a similar operation. Maybe we can just call it from here to avoid duplicate code fragments :thinking: